ui/package.json

```diff
@@ -29,7 +28,8 @@
     "react-dom": "16.8.6"
   },
   "devDependencies": {
-    "@connexta/ace": "git://github.com/connexta/ace.git#235d5b3885e1525e2f42bfbd64cd9715d85f498c",
+    "@connexta/ace": "git://github.com/connexta/ace.git#a821ea66f6ecb4404a6a23e9e86a95a0064a113f",
+    "@connexta/eslint-config-connexta": "git://github.com/connexta/eslint-config-connexta.git#3f0caece6210d25c16863d472026fbfa069521a6",
```

Comment:
We need another repo for this? Why can't it live in ace?

Reply:
afaik eslint expects shared configs to be packaged as a node module with specific naming https://eslint.org/docs/developer-guide/shareable-configs and it seemed easier to make a whole new repo